cobbler.spec

```diff
@@ -317,6 +331,12 @@ ln -sf service %{buildroot}%{_sbindir}/rccobblerd
 # cobbler-web
 rm %{buildroot}%{_sysconfdir}/cobbler/cobbler_web.conf
 
+%if 0%{?suse_version}
+# cobbler-tests
+cp -r tests/ %{buildroot}/%{_datadir}/cobbler/
+%endif
```

Comment:
This doesn't make sense to be SUSE-specific.

Reply:
I removed the SUSE specific part.